perun-openapi/openapi.yml

```diff
@@ -7372,6 +7372,20 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /urlinjsonout/usersManager/getUsersBySpecificUser:
+    post:
+      tags:
+        - UsersManager
+      operationId: getUsersBySpecificUser
+      summary: Return all users who owns the specific user.
+      parameters:
+        - $ref: '#/components/parameters/specificUserId'
+      responses:
+        '200':
+          $ref: '#/components/responses/VoidResponse'
```

Comment:
The correct response type should be '#/components/responses/ListOfUsersResponse'
Again the response type can be found in the UsersManagerMethod.java in the JavaDoc of the getUsersBySpecificUser.